Sources/ZcashLightClientKit/Block/FetchUnspentTxOutputs/UTXOFetcher.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  UTXOFetcher.swift
+//  ZcashLightClientKit
+//
+//  Created by Francisco Gindre on 6/2/21.
+//
+
+import Foundation
+
+enum UTXOFetcherError: Error {
+    case clearingFailed(_ error: Error?)
+    case fetchFailed(error: Error)
+}
+
+struct UTXOFetcherConfig {
+    let dataDB: URL
```

Comment:
this is named dataDb everywhere else. we should change every other occurrence to one format.
```suggestion
    let dataDb: URL
```